scripts/deploy.py

```diff
@@ -268,15 +278,33 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    release_version_url = (
+        'https://https://%s-dot-%s.appspot.com/library' %
+        current_release_version, APP_NAME)
```

Comment:
Will it run fine even if we are pushing the cherrypicked commits? (i.e, we have already deployed on testserver and it's serving through `oppiatestserver.appspot.com` (without version in the url))

Reply:
I think yes (not sure, we should try it I guess). But @ankita240796 the URL here has https:// twice. Is that intentional?

Reply:
I think yes too but I will need to test this once. 

https:// twice was by mistake, fixed it now.

Reply:
If I'm not mistaken, dot chars need to be treated carefully when composing URLs like this. Can we add a check/assertion at the very beginning that there is no "dot" in the current_release_version (and exit if this is false)? (Add a note at that point to say "this is required to compose the per-version URL correctly".)

Reply:
`current_release_version` is composed by replacing `.` with `-` ([Ref](https://github.com/oppia/oppia/pull/7410/files#diff-65ead24c72364b34de8e35a3249d54ffR193)), so do we still need the check?

Reply:
Yes, let's still do the check. The logic for obtaining the version as a string is separate from that for composing the release URL. Both should be handled separately and not make assumptions about each other.

Reply:
Done.

Reply:
Hi @nithusha21 @DubeySandeep, can you help me here - how should I test that cherrypicking works correctly for the demo app?

Reply:
I discussed this with @DubeySandeep and this will work correctly for cherrypicks as well.